website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -2,14 +2,14 @@
 
 ### When to use
 
-- For adding extra information, in the form of structured content, to a UI element.
-- When it’s necessary to _temporarily persist_ extra information in the UI; e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
-
 !!! Info
 
 **Temporarily persist:** By this, we mean the component will remain open so the content within can be cross-referenced with the rest of the UI until it's explicitly dismissed by the user. More details are in the [interaction](#interaction) section.
 !!!
 
+- For adding extra information, in the form of structured content, to a UI element.
+- When it’s necessary to _temporarily persist_ extra information in the UI; e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
```

Comment:
Based on the above ordering point, you could feasibly merge the content in the banner into this sentence. Not perfect (sentence is getting a bit too long), but something to consider.

```suggestion
- When it’s necessary to _temporarily persist_ extra information in the UI (the component remaining open so that the content can be cross-referenced with the rest of the UI); e.g., if the user benefits from cross-referencing information in the component with other elements in the UI.
```